app/controllers/apo_controller.rb

```diff
@@ -268,16 +286,21 @@ def update_desc_metadata
 
   def spreadsheet_template
     binary_string = RestClient.get(Argo::Config.urls.spreadsheet)
-    send_data(binary_string, :filename => 'spreadsheet_template.xlsx', :type => 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet')
+    send_data(
+      binary_string,
+      :filename => 'spreadsheet_template.xlsx',
+      :type => 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
+    )
   end
 
   private
 
   def create_obj
     raise 'missing druid' unless params[:id]
-    @object = Dor.find params[:id] # , :lightweight => true
+    @object = find_druid(params[:id])
+    return if @object.nil?
```

Comment:
now that `find_druid` would throw an `ObjectNotFoundError` if the requested object isn't found, i think these `return if @object.nil` lines are no longer necessary.